index.bs

```diff
@@ -521,34 +521,24 @@ which would have issues of its own.
 The term <dfn id="concept-device-sensor">device sensor</dfn> refers to a device's underlying
 physical sensor instance.
 
-The [=device sensors=] measure different physical quantities
-and provide corresponding <dfn>raw sensor readings</dfn>
-which are a source of information about the user and their environment.
+A [=device sensor=] measures a physical quantity
+and provides a corresponding <dfn>sensor reading</dfn>
+which is a source of information about the environment.
 
-Each [=raw sensor reading|reading=] is composed of the <dfn lt="reading value">values</dfn>
-of the different physical quantities measured by the [=device sensor|sensor=]
+Each [=sensor reading=] is composed of the <dfn lt="reading value">values</dfn>
+of the physical quantity measured by the [=device sensor=]
 at time <var ignore>t<sub>n</sub></var> which is called the <dfn>reading timestamp</dfn>.
 
-Known, <em>predictable</em> discrepancies between [=raw sensor readings=]
-and the corresponding physical quantities being measured
-are corrected through <dfn>calibration</dfn>.
-
-Known but <em>unpredictable</em> discrepancies need to be addressed dynamically
-through a process called [=sensor fusion=].
-
-[=calibration|Calibrated=] [=raw sensor readings=] are referred to as <dfn>sensor readings</dfn>,
-whether or not they have undergone [=sensor fusion=].
-
-If the [=sensor readings=] represent spatial measurements (e.g.
-acceleration, angular velocity), they must be resolved in
+If a <dfn>measurement</dfn> represents a spatial measurement (e.g.
```

Comment:
maybe better "If the sensor performs spatial measurements"

Reply:
the "measurement" definition here does not seem to explain that term, do we need this definition?

Reply:
Removed the dfn, people can refer to https://en.wikipedia.org/wiki/Measurement